README.md

```diff
@@ -1,47 +1,176 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
 
-## Installing
-`npm install --save react-e`
+## Basic Usage
 
-## Example Usage
+```javascript
+e = require('react-e')
 
-```coffeescript
-e = require 'react-e'
-
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
+e(Modal,
+  e('.container',
+    e('span.text', {className: {active: this.state.active}}, 'Some text'),
+    e(Button, {className: 'primary'}, 'Close')
+  )
+)
 ```
 Is equivalent to:
 ```jsx
-<div className="container">
-  <button className="primary">Click me</button>
-  <span className="text active">Some text</span>
-</div>
+<Modal>
+  <div className="container">
+    <span className={'text' + (this.state.active ? ' active' : '')}>Some text</span>
+    <Button className='primary'>Close</Button>
+  </div>
+</Modal>
 ```
 
-## Use with css modules
 
-```coffeescript
-styles = require 'index.css'
-###
-styles =
-  container: 'local1'
-  primary: 'local2'
-  text: 'local3'
-###
+## Installing
+`npm install --save react-e`
+
+
+## Features
+
+React-e is a drop in replacement for `react.createElement`, but it provides syntactic advantages:
+
+#### `props` can be omitted even if you have children:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre lang="javascript">
+e('div',
+  e('span', 'Hello world'))
+    </pre></td>
+    <td><pre lang="javascript">
+createElement('div', {},
+  createElement('span', {}, 'Hello world'))
+    </pre></td>
+    <td><pre lang="jsx">
+&lt;div&gt;
+  &lt;span&gt;Hello world&lt;/span&gt;
+&lt;/div&gt;
+    </pre></td>
+  </tr>
+</table>
+
+#### static classnames can be provided inline using jade syntax:
+<table>
+  <tr>
+    <th>react-e example</th>
```

Comment:
Thoughts about naming this "react-e version"? I think you are providing 3 different versions of the same example here.